Documentation updates for v1.2 release

dkobras opened the pull request.

michael-o commented on ChangeLog:
```diff
@@ -1,3 +1,17 @@
+Release 1.2rc1
```
Do we really need an RC?

michael-o replied:
I think we can go without and push 1.2.1 if necessary.

michael-o commented on ChangeLog:
```diff
@@ -1,3 +1,17 @@
+- New co-maintainer Michael Osipov
+- Improvement: allow to delete and reset computer account
+- Improvement: prefer SASL mechanism GSS-SPNEGO over GSSAPI for LDAP
+               connections to domain controllers (thanks, James Ralston!)
```
I still don't understand the benefit of the change. In fact, I have commented on the commit, but the author did not respond to it.

dkobras replied:
I've added a comment there. Hope this clarifies the motivation of this change.

michael-o reviewed (changes requested):
From `git log --pretty=oneline  v1.1rc1...master` I miss:
```
Fix for Heimdal: Keyblock (#119)
improve IPv6 support (#129)
Implements --use-samba-cmd option (#133)
Initialize struct addrinfo in a portable manner (#161) (#165)
Print effectively supported encryption types when constructing a keytab
Clean up MS Kerberos encryption types (#157) (#168)
Clean up --remove-enctype option parsing + Fix --remove-enctype option description in manpage
always add host/shortname in addition to host/longname
Provide gzipped release tarball as well (#132)
```

dkobras commented:
> From `git log --pretty=oneline v1.1rc1...master` I miss:
> 
> ```
> Fix for Heimdal: Keyblock (#119)
> improve IPv6 support (#129)
> Implements --use-samba-cmd option (#133)
> Initialize struct addrinfo in a portable manner (#161) (#165)
> Print effectively supported encryption types when constructing a keytab
> Clean up MS Kerberos encryption types (#157) (#168)
> Clean up --remove-enctype option parsing + Fix --remove-enctype option description in manpage
> always add host/shortname in addition to host/longname
> Provide gzipped release tarball as well (#132)
> ```

I've just added a few items for the logging and documentation updates. From your list above, some of them are already covered. The rest should not contain any user-visible changes, as far as I can tell. (But please feel free to add more entries if you disagree.)

michael-o commented:
Guys, can we complete this PR by end of month?

dkobras commented:
@michael-o I've rebased the PR, and dropped the `rc1` from the version number. The commit `always add host/shortname in addition to host/longname` you mentioned as missing from the changelog was just a fixup in a larger PR. The overall change (`consistent qualification of default SPN entries`) is documented already. For me, it's good to go like this.

michael-o reviewed (approved):
This is now complete for now. It can be squashed, rebased and merged into master.

michael-o commented:
> @michael-o I've rebased the PR, and dropped the `rc1` from the version number. The commit `always add host/shortname in addition to host/longname` you mentioned as missing from the changelog was just a fixup in a larger PR. The overall change (`consistent qualification of default SPN entries`) is documented already. For me, it's good to go like this.

Accepted, makes sense.

dkobras commented:
@michael-o Ok, I'll rebase once more to get rid of the revert in the history, and then do the merge.

michael-o commented:
> @michael-o Ok, I'll rebase once more to get rid of the revert in the history, and then do the merge.

Please squash/fixup and merge then.
